### DISCUSSION TITLE
Firefox CSS

### DISCUSSION THREAD

**Discussion by ghost** [catppuccin/catppuccin#1958] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

Firefox supports theming via `userChrome.css` and `userContent.css`, and people have made some [pretty cool themes](https://github.com/manilarome/blurredfox#blurredfox) using them.

### Any additional comments?

As opposed to the [Catppuccin Theme for Firefox](https://github.com/catppuccin/firefox#catppuccin-for-firefox), this can override the browsers appearance, layout, etc., rather than just the UI

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by justTOBBI** [comment]
We already have a userchrome section for firefox in the repo. Feel free to add yours too :)

**Reply by backwardspy** [reply]
reference: https://github.com/catppuccin/firefox/blob/main/docs/FirefoxCSS.md

**Reply by ghost** [reply]
@backwardspy Can I install colors, without using the cascade UI?

**Reply by backwardspy** [reply]
@abrendtro i think it would need to be custom userchrome. as tobbi said, you're more than welcome to add it to that list when it's done!

**Reply by ghost** [reply]
@backwardspy I can also tell firefox to "Use system colors", which follows the GTK+ theme

**Comment by sgoudham** [comment]
This should be raised on the [catppuccin/firefox](https://github.com/catppuccin/firefox) repository itself.
